test/integration/glue_test.py

```diff
@@ -73,56 +74,56 @@ def save_to_devtools_glue(df, test_dir, request):
         "utilities_ferc1_dbf",
         ["utility_id_ferc1"],
         None,
-        id="validate_utility_id_ferc1_in_utilities_ferc1_dbf",
+        id="missing_utility_id_ferc1_in_utilities_ferc1_dbf",
     ),
     pytest.param(
         "utilities_ferc1",
         "utilities_ferc1_xbrl",
         ["utility_id_ferc1"],
         None,
-        id="validate_utility_id_ferc1_in_utilities_ferc1_xbrl",
+        id="missing_utility_id_ferc1_in_utilities_ferc1_xbrl",
     ),
     pytest.param(
         "utilities_ferc1",
         "plants_ferc1",
         ["utility_id_ferc1"],
         None,
-        id="validate_utility_id_ferc1_in_plants_ferc1",
+        id="missing_utility_id_ferc1_in_plants_ferc1",
     ),
     pytest.param(
         "utilities_ferc1_xbrl",
         "util_ids_ferc1_raw_xbrl",
         ["utility_id_ferc1_xbrl"],
         None,
-        id="check_for_unmmaped_utility_id_ferc1_xbrl_in_raw_xbrl",
+        id="missing_utility_id_ferc1_xbrl_in_raw_xbrl",
     ),
     pytest.param(
         "utilities_ferc1_dbf",
         "util_ids_ferc1_raw_dbf",
         ["utility_id_ferc1_dbf"],
         None,
-        id="check_for_unmmaped_utility_id_ferc1_dbf_in_raw_dbf",
+        id="missing_utility_id_ferc1_dbf_in_raw_dbf",
     ),
     pytest.param(
         "plants_pudl",
         "plants_ferc1",
         ["plant_id_pudl"],
         None,
-        id="validate_plant_id_pudl_in_plants_ferc1",
+        id="missing_plant_id_pudl_in_plants_ferc1",
     ),
     pytest.param(
         "plants_ferc1",
         "plants_ferc1_raw",
         ["utility_id_ferc1", "plant_name_ferc1"],
         None,
-        id="check_for_unmmapped_plants_in_plants_ferc1",
+        id="missing_plants_in_plants_ferc1",
     ),
     pytest.param(
         "plants_eia",
         "plants_eia_pudl_db",
         ["plant_id_eia"],
         document_plant_eia_ids_for_manual_mapping,
```

Comment:
Does `document_plant_eia_ids_for_manual_mapping()` work when it receives a `pd.Index`? The call signature currently says it expects a `pd.Series`.